CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Removed
 
+## [v4.9.2] - 2018-03-13
+
+### Fixed
+
+- Another tweak to search request debouncing
```

Comment:
s/to/for?

Reply:
I think both are correct. I meant it as in "We applied another tweak to the […] debouncing".